### DISCUSSION TITLE
Modify the font size of the Shape's Sidebar

### DISCUSSION THREAD

**Discussion by FedericoDellAquila** [jgraph/drawio-desktop#1690] [Q&A]
Hello everyone! I would like to know if there's a solution for my problem. I have a 3840 x 2160 monitor, and when I open Draw.io I struggle to read the name of even tell each shape apart:

![drawio](https://github.com/jgraph/drawio-desktop/assets/22677681/ed29fddc-c348-40a3-afe0-d5b3b6eaf58d)

Is there anything I can do to modify the size of the image and the name text? If there's any other info that you need to help, I'll be more than happy to provide it. Thanks to everyone in advance.

**Comment by alderg** [comment]
Try the following via Extras, Configuration:
```
{
  "sidebarTitleSize": 12,
  "thumbWidth": 42,
  "thumbHeight": 42
}
```

**Reply by FedericoDellAquila** [reply]
![image](https://github.com/jgraph/drawio-desktop/assets/22677681/e357024c-a35d-4919-8fa8-3ead874c73fa)

Thanks, that worked! I tried to use these values before, but my mistake was that I was adding them in the Configuration->Preferences rather than just in Configuration.

One last thing, even if I believe this one would be a little too much to ask: would it be possible to define the values relative to the monitor resolution? This would be nice to keep everything proportionate independently by the monitor.
